src/main/java/io/weaviate/client6/v1/api/backup/Backup.java

```diff
@@ -34,7 +34,7 @@ public record Backup(
     /** Time at which the backup was completed, successfully or otherwise. */
     @SerializedName("completedAt") OffsetDateTime completedAt,
     /** Backup size in GiB. */
-    @SerializedName("size") Integer sizeGiB,
+    @SerializedName("size") Double sizeGiB,
```

Comment:
Can we use `Float` here? I don't think the size will ever exceed `3.4028235+E38 Gb`